src/sentry/api/endpoints/organization_events_root_cause_analysis.py

```diff
@@ -190,14 +193,15 @@ def get(self, request, organization):
 
         regression_breakpoint = parse_datetime_string(regression_breakpoint)
 
-        params = self.get_snuba_params(request, organization)
+        snuba_params, _ = self.get_snuba_dataclass(request, organization)
 
         with handle_query_errors():
             transaction_count_query = metrics_query(
                 ["count()"],
                 f'event.type:transaction transaction:"{transaction_name}"',
-                params,
+                params={},
                 referrer=BASE_REFERRER,
+                snuba_params=snuba_params,
```

Comment:
nit: move `snuba_params` next to `params`

Reply:
:+1: I'll do that in a follow up